src/modules/site-v2/templates/tools/phenotype/report.html

```diff
@@ -0,0 +1,58 @@
+{% extends "_layouts/default-nocontainer.html" %}
+
+{% block custom_head %}
+
+<!-- D3js -->
+<script src="//d3js.org/d3.v4.min.js"></script>
+
+<style>
+    .tooltip {
+        position: absolute;
+        font-size: 12px;
+        width:  auto;
+        height: auto;
+        border: 1px solid black;
+        pointer-events: none;
+        line-height: 1px;
+        background-color: white;
+    }
+</style>
+
+{% endblock %}
+
+{% block content %}
+
+<div class="container-fluid px-5">
+    <p><i>Spearman Correlation = {{ result.correlation|round(4) }}</i></p>
+    <p><i>P-value = {{ result.p_value|round(4) }}</i></p>
+    <div id="phenotype-chart"></div>
+</div>
+
+{% endblock %}
+
+
+{% block script %}
+<script src="{{ url_for('static', filename='js/scatter-histogram.js') }}"></script>
+
+<script>
+
+const data = {{ data.trait_values|tojson }};
+
+scatterplot_histograms('#phenotype-chart', data, {
+    hist_height:   60,
+    width:         600,
```

Comment:
second order priority -- we might need to adjust width/height for mobile devices. (The first priority is to get the laptop version).

Reply:
Agreed -- I imagine we'll want this to choose this value based on window width (or maybe even make the graph size responsive).  I've just filled in reasonable values for now, and still have to work w Natalie to lay out the graph(s) on the report page.